custom_components/waste_collection_schedule/waste_collection_schedule/source/reading_gov_uk.py

```diff
@@ -0,0 +1,68 @@
+from datetime import date, datetime
+from typing import List
+
+import requests
+from waste_collection_schedule import Collection
+
+TITLE = "Reading Council"
+DESCRIPTION = "Source for reading.gov.uk services for Reading Council"
+URL = "https://reading.gov.uk"
+TEST_CASES = {
+    "known_uprn": {"uprn": "310027679"},
+    "known_uprn as number": {"uprn": 310027679},
+    "unknown_uprn_by_number": {"postcode": "RG31 5PN", "housenumberorname": "65"},
+    "unknown_uprn_by_number as number": {"postcode": "RG31 5PN", "housenumberorname": 65}
+}
+
+ICONS = {
+    "Rubbish": "mdi:trash-can",
+    "Recycling": "mdi:recycle",
+    "Food Waste": "mdi:food",
+    "Garden Waste": "mdi:tree"
+}
+
+COLLECTIONS = {
+    "Domestic Waste Collection Service": "Rubbish",
+    "Recycling Collection Service": "Recycling",
+    "Food Waste Collection Service": "Food Waste",
+    "Garden Waste Collection Service": "Garden Waste"
+}
+
+SEARCH_URLS = {
+    "UPRN": "https://api.reading.gov.uk/rbc/getaddresses",
+    "COLLECTION": "https://api.reading.gov.uk/api/collections"
+}
+
+
+class Source:
+    def __init__(
+        self, uprn=None, postcode=None, housenumberorname=None
```

Comment:
`housenumberorname` is not the same as defined in the md file, so example configuration provided by md file would not work

Reply:
Fixed